crates/biome_deserialize/src/lib.rs

```diff
@@ -142,6 +143,9 @@ pub trait DeserializableValue: Sized {
 
     /// Returns the type of this value.
     fn visitable_type(&self) -> Option<DeserializableType>;
+
+    /// Returns a JSON representation of the value, if possible.
+    fn into_json_value(self) -> Option<AnyJsonValue>;
```

Comment:
I am really not fond of adding this method. This breaks a bit the overall abstraction.
I noted that the method is only used once. I don't understand why we need to handle raw json?

Reply:
Neither do I, things are a bit blurry to be honest.

Reply:
Removed it!